packages/cli/src/services/frontend.service.ts

```diff
@@ -362,7 +362,8 @@ export class FrontendService {
 
 		this.settings.enterprise.projects.team.limit = this.license.getTeamProjectLimit();
 
-		this.settings.folders.enabled = config.get('folders.enabled');
+		this.settings.folders.enabled =
+			config.get('folders.enabled') || this.license.isFoldersEnabled();
```

Comment:
I think we still want to keep the feature behind the feature flag so I would use `&&` instead of `||` to prevent it from being enabled for existing registered community users before we release it.